api/python/cellxgene_census/tests/experimental/pp/test_hvg.py

```diff
@@ -31,26 +36,46 @@
         ),
         pytest.param(
             "mus_musculus",
-            'is_primary_data == True and assay == "Smart-seq"',
+            'is_primary_data == True and assay == "Smart-seq2"',
             marks=pytest.mark.expensive,
         ),
     ],
 )
-@pytest.mark.parametrize("n_top_genes", (5, 500))
+@pytest.mark.parametrize("n_top_genes", (50, 500))
 @pytest.mark.parametrize(
     "batch_key",
     (
         None,
         "dataset_id",
         ["suspension_type", "assay_ontology_term_id"],
-        ("suspension_type", "assay_ontology_term_id", "dataset_id"),
-        ["dataset_id", "assay_ontology_term_id", "suspension_type", "donor_id"],
+        pytest.param(
+            ("suspension_type", "assay_ontology_term_id", "dataset_id"),
+            marks=pytest.mark.expensive,
+        ),
+        pytest.param(
+            ["dataset_id", "assay_ontology_term_id", "suspension_type", "donor_id"],
+            marks=pytest.mark.expensive,
+        ),
+    ),
+)
+@pytest.mark.parametrize(
+    "span",
+    (
+        pytest.param(None, marks=pytest.mark.expensive),
+        0.5,
+    ),
+)
+@pytest.mark.parametrize(
+    "version",
+    (
+        "latest",
+        pytest.param("stable", marks=pytest.mark.expensive),
```

Comment:
"latest" is just as expensive as "stable", right? I assume we're just trying to run just one for "non-expensive" mode then.

Reply:
yes, correct.  If there is a better way, LMK.  Our daily CI unit test was getting too long running.